llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -2085,6 +2085,12 @@ MemoryDepChecker::getDependenceDistanceStrideAndSize(
   if (!isa<SCEVConstant>(Dist))
     FoundNonConstantDistanceDependence |= StrideAPtrInt == StrideBPtrInt;
 
+  // If distance is a SCEVCouldNotCompute, return Unknown immediately.
+  if (isa<SCEVCouldNotCompute>(Dist)) {
```

Comment:
It looks like we are no longer performing `CheckCompletelyBeforeOrAfter` as before, so we can't come up with `NoDep` as we used to. So it's not entirely NFC, or am I missing something?

Reply:
Ah, yes indeed. The only functional difference is that we'll always get this debug message now:

```
LLVM_DEBUG(dbgs() << "LAA: Dependence because of uncomputable distance.\n");
```

Even if the result is `NoDep`. I guess it doesn't matter?

Reply:
I've tweaked the debug message now.